scripts/reset-webhook-queue.sh

```
@@ -0,0 +1,69 @@
+if [ "$total_jobs" -eq 0 ]; then
+  echo "✅ No jobs found in queue. Nothing to clean."
+  exit 0
+fi
+
+echo ""
+echo "⚠️  WARNING: This will delete ALL webhook indexing jobs from the queue!"
+echo "   - $started_length active jobs will be terminated"
+echo "   - $queue_length waiting jobs will be removed"
+echo "   - All job history will be lost"
+echo ""
+read -p "Continue? (y/N): " confirm
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)